string dmp_name
geometry_msgs/PoseStamped initial_pose
geometry_msgs/PoseStamped goal_pose
float64 tau
float64 dt
---
string result